example/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2014 - 2023 Helmholtz-Zentrum Dresden - Rossendorf e.V. (HZDR)
```

Comment:
Since alpaka started out as a HZDR research project I believe the formal copyright holder is HZDR. Please correct me if I'm wrong @alpaka-group/alpaka-maintainers.

Reply:
I don't think this is right. This makes it look like the entire alpaka examples are copyrighted by HZDR. That is at least wrong for all LOCs I have written, which legally belong to CERN.

Why do you even need to put a LICENSE file here?

Reply:
We should discuss the copyright issue in tomorrow's VC.

> Why do you even need to put a LICENSE file here?

We need to put the ISC license for the examples somewhere if we delete the header comments. Feel free to suggest a better location.

Reply:
> We need to put the ISC license for the examples somewhere if we delete the header comments. Feel free to suggest a better location.

Do we need to do that? The ISC license doesn't state that a copy of the license must be included with the source code. We could just have the ISC SPDX license identifier in the example files and be good I think.